lib/model.js

```diff
@@ -169,6 +195,14 @@ class Model {
 
 		this.useReadDB = false;
 
+		if(this.session && this.session.userId && isObject(values)) {
+
+			values = {
```

Comment:
No hace falta el spread en este caso:
`values.userModified = this.session.userId`